sycl/test-e2e/GroupAlgorithm/root_group.cpp

```diff
@@ -3,6 +3,9 @@
 // RUN: %{build} -I . -o %t.out
 // RUN: %{run} %t.out
 
+// Disabled temporarily while investigation into the failure is ongoing.
+// UNSUPPORTED: gpu-intel-dg2
```

Comment:
If the failure is reported in open-source, it would be good to add a GH issue/discussion link.

Reply:
It is an internal tracker, but otherwise I definitely would have. 😉

Reply:
> It is an internal tracker, but otherwise I definitely would have. 😉

We can always have a GH issue, even if the tracker was submitted internally. I would vote for always having a GH issue, because without it is it not clear if there is a tracker at all